config/jobs/image-pushing/k8s-staging-nfd-operator.yaml

```diff
@@ -15,6 +15,7 @@ postsubmits:
       # job makes sense on every branch (unless it's setting a `latest` tag it
       # probably does).
       branches:
+        - ^main$
```

Comment:
nit 🙄: unrelated change

Reply:
plop why this change is here, I had a diff branch 🤔  jjejeje sorry fixing....

Reply:
np 😊